enzyme/test/Integration/blasinfra.h

```diff
@@ -513,6 +588,30 @@ void cblas_dgemm(char layout, char transA, char transB, int M, int N, int K, dou
                                 M, N, K, lda, ldb, ldc});
 }
 
+using cublasOperation_t::CUBLAS_OP_UNUSED;
+
+__attribute__((noinline))
+cublasStatus_t cublas_ddot(cublasHandle_t *handle, int N, double* X, int incx, double* Y, int incy, double* result) {
+    CuBlasCall call = {inDerivative, CallType::DOT,
```

Comment:
I would probalby not make a separate cucalls, but add an extra arg to calls saying device or API

Reply:
I did consider / tried that for a while, but it turned out to be a bit annoying for a few reasons.
Let me try this one first, if I run into stronger issues or later I can still merge it?